config-linux.md

```diff
@@ -455,6 +455,39 @@ The following parameters can be specified to set up the controller:
    }
 ```
 
+### <a name="configLinuxRDMA" />RDMA
+
+**`rdma`** (object, OPTIONAL) represents the cgroup subsystem `rdma`.
+For more information, see the kernel cgroups documentation about [rdma][cgroup-v1-rdma].
+
+The following parameters can be specified to set up the controller:
+
+* **`limits`** *(list of objects, OPTIONAL)* - specifies a list of objects of the limits applicable to processes in the group.
+    * **`hca_device`** *(string, REQUIRED)* - specifies the device name whose resources limit to be configured
+        The following parameters can be specified per-device:
+        * **`hca_handles`** *(uint32, OPTIONAL)* - specifies the maximum number of hca_objects in the cgroup
+        * **`hca_objects`** *(uint32, OPTIONAL)* - specifies the maximum number of hca_handles in the cgroup
+        You MUST specify at least one of the `hca_handles` or `hca_objects` in a given entry, and MAY specify both.
+
+#### Example
+
+```json
+    "rdma": {
+        "limits": [
+            "mlx5_1": {
```

Comment:
This is not valid JSON.  I think you want to drop `limits` and just make [`rdma` an object with device-name keys and handles/objects-limit values][1].

[1]: https://github.com/opencontainers/runtime-spec/pull/942#discussion_r155913558